chat-ui/src/DevCenterChatbot.tsx

```diff
@@ -41,11 +46,26 @@ const styles = {
       right: 49px;
     }
   `,
-  message_prompts: css`
+  message_prompts: (duration: number) => css`
```

Comment:
Are you planning on having multiple durations? If not, instead of using a function we could move `duration` up to be a root-level variable.